vmware/vsphere_provider.yml

```diff
@@ -26,4 +26,4 @@ spec:
   settings:
     vddkInitImage: 'quay.io/lrotenbe/vddk:7.0.3'
```

Comment:
can we do such thing? need to check how CDI did this.. can it work without vddk? can we use the image CDI use, if they use vddk in their CI

Reply:
yeah, my concern is about the licensing of vddk - it if permits publishing the image like that